website/docs/r/cosmosdb_postgresql_cluster.html.markdown

```diff
@@ -49,6 +47,8 @@ The following arguments are supported:
 
 * `node_count` - (Required) The worker node count of the Azure Cosmos DB for PostgreSQL Cluster. Possible value is between `0` and `20` except `1`.
 
+* `administrator_login_password` - (Optional) The password of the administrator login.
```

Comment:
The documentation requires an update too 
```suggestion
* `administrator_login_password` - (Optional) The password of the administrator login. This is required when `source_resource_id` is not set.
```

Reply:
Updated